app/src/main/java/com/sopt/now/data/model/response/ResponseFriendsDto.kt

```diff
@@ -0,0 +1,43 @@
+package com.sopt.now.data.model.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseFriendsDto(
+    @SerialName("page")
+    val page: Int,
```

Comment:
```suggestion
    val page: Int = -1,
```
기본값 설정해주면 더 좋을 것 같아요